src/core/ucc_sbgp.h

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020-2021.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+#ifndef UCC_SBGP_H_
+#define UCC_SBGP_H_
+#include "ucc/api/ucc.h"
+#include "utils/ucc_datastruct.h"
+#include "utils/ucc_coll_utils.h"
+
+typedef enum ucc_sbgp_type_t {
+    UCC_SBGP_UNDEF = 0,
+    UCC_SBGP_NUMA,
```

Comment:
While most of them are the property of where control resides (SOCKET, NODE), except NUMA. Did you find use cases for them ?  How do you plan to create groups for it -  find the nearest threads to the NUMA domain and group them?

Reply:
Can you mix say SOCKET AND NUMA?

Reply:
discussed in WG